cli/src/config/templates.toml

```diff
@@ -41,10 +41,24 @@ tag_list = '''
 label("tag", name) ++ format_ref_targets(self) ++ "\n"
 '''
 
+# TODO: "op_log" is not a great label name for op_summary. We could replace it
+# with `operation` and insert this new label throughout the op log templates.
```

Comment:
Yes. Maybe we can insert `op_summary`/`commit_summary` labels if needed. That's the current convention.

Reply:
That sounds good. I'd prefer to leave this cleanup to some later time, though.

Reply:
You mean that `format_operation` and related macros would also add the `op_summary` label? I thought we'd want one label to be used both there and in `op_summary` so that we don't need to add many more colors to `colors.toml`.

Ultimately, either is fine by me, but if I understand the plan, I can adjust the TODO accordingly.

Reply:
I mean the label should be inserted by caller, not by template. (See `cmd_op_log()` for instance.)

Anyway, if we remove `"finished "`, I think we can remove `label("op_log", ...)` too.

Reply:
> Anyway, if we remove "finished ", I think we can remove label("op_log", ...) too.

I don't follow. Without `op_log`, the labels from `colors.toml` aren't triggered and there is no color at all. Somehow, there has to be a tag that says that we have an *operation* id here, as opposed to a *change* id or a *commit* id.

Reply:
> > Anyway, if we remove "finished ", I think we can remove label("op_log", ...) too.
> 
> I don't follow. Without `op_log`, the labels from `colors.toml` aren't triggered and there is no color at all.

Oh, I didn't notice this was the fix for the existing problem. Inserting `operation` label somewhere sounds also good to me (to denote the `self` type is `Operation`.)

Reply:
Yeah, I had a hard time stopping at the bug fix when I saw other things that could be better.

I belatedly split this into two commits.